x-pack/plugins/maps/public/classes/util/is_refresh_only_query.ts

```diff
@@ -5,20 +5,16 @@
  * 2.0.
  */
 
-import { MapQuery } from '../../../common/descriptor_types';
+import type { Query } from 'src/plugins/data/common';
 
 // Refresh only query is query where timestamps are different but query is the same.
 // Triggered by clicking "Refresh" button in QueryBar
 export function isRefreshOnlyQuery(
```

Comment:
isRefreshOnlyQuery function serves no purpose now that `queryLastTriggeredAt` has been removed from `query`. How about removing it?

This will also effect tiled_vector_layer which uses this function to set a new uuid vs re-using previous URL token. This logic will need to reworked with isForceRefresh instead of isRefreshOnlyQuery?